src/entities/user-session.entity.ts

```diff
@@ -0,0 +1,19 @@
+import { BaseEntity, Column, Entity, PrimaryColumn } from 'typeorm';
```

Comment:
this should not be needed once you remove CDX services in exchange for the Auth API